<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiGestionGenre</class>
 <widget class="QWidget" name="GuiGestionGenre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>320</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gestion des genres</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitreGenre">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Liste des genres :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tvGenre"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAjouter">
         <property name="text">
          <string>Ajouter</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/add2.png</normaloff>:/SeboApp/Resources/add2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnModifier">
         <property name="text">
          <string>Modifier</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/modifier2.png</normaloff>:/SeboApp/Resources/modifier2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSupprimer">
         <property name="text">
          <string>Supprimer</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/delete2.png</normaloff>:/SeboApp/Resources/delete2.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../SeboApp.qrc"/>
 </resources>
 <connections/>
</ui>
